pat_helper.py

```diff
@@ -194,8 +199,9 @@ def revoke_pat():
     PAT = 'revoked'
 
 def print_outputs():
-    print(f"::set-output name=pat::${PAT}")
-    print(f"::set-output name=pat_id::${PAT_ID}")
+    print(f"::set-output name=username::{USERNAME}")
```

Comment:
needed for revoking I guess ?

Reply:
Yeah. We have defined the PAT_ID as an output in the metadata, but haven't set it in the script. 

Also removing the `$` character which messed up the outputs.